pyccel/codegen/printing/fcode.py

```diff
@@ -651,6 +657,41 @@ def _print_MathFloor(self, expr):
         prec_code = self._print(prec)
         return 'floor({}, kind={})'.format(arg_code, prec_code)
 
+    # --------------------------- pyc_math functions --------------------------
+
+    def _print_MathFactorial(self, expr):
+        """ Print the equivalent code to pycthon factorial function call"""
```

Comment:
Typo: pycthon -> python

This is repeated multiple times in the file.

Reply:
fixed in both fcode and ccode